hw/xbox/amd_smbus.c

```diff
@@ -37,7 +37,7 @@
 #endif
 
 /* AMD756 SMBus address offsets */
-#define SMB_ADDR_OFFSET     0xE0
+#define SMB_ADDR_OFFSET 0xE0
 #define SMB_IOSIZE      16
```

Comment:
nit: I'd probably remove alignment as it's mixing hex and dec.

Reply:
I don't understand the argument?

Reply:
If you align these, it's easier to accidentally use them in the wrong way (such as using 237384 instead of 0x237384 and vice versa):

```c
#define FOO 0x237384
#define ABC 10341238
#define BAR 0x569473
```

While this does not apply as much here, it can still be an issue.
So to avoid this, I typically avoid aligning different number bases altogether.
We *do* have this issue in the NV2A clock constants.

*(But I also like to keep offsets in hex, and most sizes in decimal by the way)*